life/modernized/26-chapter-26.adoc

```diff
@@ -1591,6 +1685,10 @@ for the virtues of Gulielma Maria Penn.
 But the best account of her character and end, is given by her husband,
 which containing several edifying passages, is here inserted.
 
+[.embedded-content-document.letter]
+--
+
+[.salutation]
 My dear wife,
 after eight months illness (though she never perfectly
```

Comment:
Where it says "My dear wife" here, its not a salutation. It's the beginning of a sentence... where he goes on to say "My dear wife... departed this life...".  You did it right in the original.  [btw... if it WERE a salutation (which its not), there would need to be blank line under it.]

Reply:
Oh man, I went back into this and looked at the PDF because I was like, "how could I have missed that when I was comparing them??" but even with the [salutation] it made it look just like normal so that's probably why I didn't catch it when comparing the two PDF's. Sorry about that!